docs/content/users/quickstart.md

```diff
@@ -541,20 +542,21 @@ The Laravel project type can be used for [StarterKits](https://laravel.com/docs/
     ddev config --project-type=laravel --docroot=public
     ddev start
     ddev composer install
-    ddev php artisan key:generate
+    ddev composer run-script post-root-package-install
+    ddev composer run-script post-create-project-cmd
     ddev launch
     ```
 
 !!!tip "Want to use a SQLite database for Laravel?"
     DDEV defaults to using a MariaDB database to better represent a production environment.
 
-    To select the [Laravel 11 defaults](https://laravel.com/docs/11.x/releases#application-defaults) for SQLite, use this command for `ddev config`:
+    To select the [Laravel defaults](https://laravel.com/docs/11.x/releases#application-defaults) for SQLite, use this command for `ddev config`:
     ```bash
     ddev config --project-type=laravel --docroot=public --omit-containers=db --disable-settings-management=true
     ```
 
```

Comment:
A problem with this section is that only applies when starting a new project.
For existing projects you need to:

- update `.env` to `DB_CONNECTION="sqlite"`

```bash
ddev config --project-type=laravel --docroot=public --omit-containers=db --disable-settings-management=true
ddev restart
ddev artisan migrate:fresh
 ```

If you don't update the `.env` you will get the following error (included for searchability): 

```
SQLSTATE[HY000] [2002] php_network_getaddresses: getaddrinfo for db failed: Name or service not known (Connection: mariadb, SQL: select * from `sessions` where `id` = jvwGaqOpdMyrsUZa412ljRqpuaNOIqPSw2NJE2XN limit 1)
```

Reply:
Nice catch!

I think we should put the SQLite config on a separate tab, where we mention how to migrate from one to the other.

If we add everything on one tab, it will complicate things for newcomers.